src/py/flwr/server/history.py

```diff
@@ -73,6 +73,20 @@ def add_metrics_centralized(
             self.metrics_centralized[key].append((server_round, metrics[key]))
 
     def __repr__(self) -> str:
+        """Create a representation of History.
+
+        The representation consists of the following data (for each round) if present:
+        * distributed loss,
+        * centralized loss,
+        * distributed training metrics,
+        * distributed evaluation metrics,
+        * centralized metrics.
```

Comment:
```suggestion
        * distributed loss
        * centralized loss
        * distributed training metrics
        * distributed evaluation metrics
        * centralized metrics
```

Did we use this style somewhere else in bullet lists? Otherwise, I suggest removing the "," and ".".

Wdyt @adam-narozniak ?

Reply:
Yes, but I haven't checked it while adding. We used dots at the end of each bullet point
https://github.com/adap/flower/blob/34e9a4e889c95e6607b11b0ba0dcaa48f2735e24/src/py/flwr/driver/app.py#L77-L82
And here
https://github.com/adap/flower/blob/34e9a4e889c95e6607b11b0ba0dcaa48f2735e24/src/py/flwr/server/fleet/grpc_bidi/grpc_server.py#L106-L112
Should I move to the dots?

Reply:
Yes, additionally, add the new line I would suggest. Let's keep it consistent.

Reply:
Done - "." at the end of each bullet point and the new line between the ":" and the first bullet point.